mti_nma/bin/all.py

```diff
@@ -102,9 +118,10 @@ def run(
                     local_directory=str(log_dir),
                     log_directory=str(log_dir)
                 )
+                log.info("Created SLURMCluster")
 
-                # Scale workers
-                cluster.adapt(minimum_jobs=1, maximum_jobs=40)
+                # Set adaptive worker settings
+                cluster.scale_up(40)
```

Comment:
I would change this to
```suggestion
                cluster.adapt(minimum_jobs=30, maximum_jobs=60)
```